crates/binjs_io/src/entropy/read/prelude_decoders.rs

```diff
@@ -163,25 +163,14 @@ where
 
         // Extract slice.
         let mut buf = Vec::with_capacity(byte_len); // FIXME: We may want some maximal size here.
-        unsafe {
-            buf.set_len(byte_len);
-        }
+        buf.resize(byte_len, 0);
```

Comment:
Nit: this and the former line can be combined into `vec![0; byte_len]` (as a bonus to clarity, it's also slightly more efficient as it can use `calloc`).

Reply:
Ooooohhh...
I had never heard of this syntax, thanks!